packages/web/src/assetImports.ts

```diff
@@ -6,6 +6,8 @@
 // see: redwood/packages/core/config/webpack.common.js
 // These declarations are the most common types
 
+// TODO(jgmw): I doubt these comments above are still correct
```

Comment:
The type defs are still correct, but comments aren't. 

But ideally we'd just be able to reference the Vite asset import style without having to specify it

```
/// <reference types="vite/client" />
```

https://vitejs.dev/guide/features.html#client-types